### PR TITLE
10.0 readme updates

### PR DESCRIPTION
**Pull request by neilpeterson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +52,63 @@ Snippets for deployment template and 70+ Azure resources.
...
+
+![Inserting an item into an Azure Resource Manager template](./images/insert-item-outline.png)
+
+## Other features
+
+  - [Signature help](https://code.visualstudio.com/docs/editor/editingevolved#_parameter-hints) for TLE function parameters
+  - [Go To Definition](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-definition) for variable and parameter references
+  - [Peek](https://code.visualstudio.com/docs/editor/editingevolved#_peek) for variable and parameter definitions
+  - Find all references (Shift + F12) for variables and parameters
+  - Rename all references (F2) for variables and parameters
```
Comment (StephenWeatherford) [inline README.md:71]:
```suggestion
  - Rename (F2) variables and parameters and their references
```

File: README.md [hunk 2]
```diff
@@ -21,29 +21,20 @@ The Azure Resource Manager (ARM) Tools for Visual Studio Code provides language
...
   - [resourceGroup() properties](https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-functions/#resourcegroup)
   - [subscription() properties](https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-functions/#subscription)
+  - [resourceId() function](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#resourceid)
   - Properties of references to variables that are objects
 
   ## Parameter files
 
   - Associate a parameter file with a template for full validation of both
   - Open associated parameter file from the template
   - Create a new parameter file from parameters defined within a template
```
Comment (StephenWeatherford) [inline README.md:31]: Do we need instructions for creating a new param file?  Should it be given better visibility in the readme?

File: README.md (hunk 1 above)
Comment (neilpeterson) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (neilpeterson) [reply README.md]: Agreed. I've added a little bit and also added a link to the [VS Code tutorial](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/quickstart-create-templates-use-visual-studio-code?tabs=CLI) which includes this functionality. I would be keen on considering another re-write of this readme that is more action > outcome oriented but will do so for a future release.

File: README.md [hunk 3]
```diff
@@ -54,21 +45,31 @@ Snippets for deployment template and 70+ Azure resources.
 
 ## Insert Item
 
-In addition to adding snippets from the code editor, the Insert Item feature can be used to insert new parameters, user-defined functions, variables, resources, and outputs. To do so right-click on the template in the code editor, select **Insert Item**, and follow the on-screen prompts.
+In addition to adding snippets from the code editor, the Insert Item feature can be used to insert new parameters, user-defined functions, variables, resources, and outputs. To do so right-click on the template in the code editor, select **Insert Item** and follow the on-screen prompts.
 
 ![Inserting an item into an Azure Resource Manager template](./images/insert-item.png)
 
-You can also right-click on any element in the ARM Template Outline to initiate the insert item process.
+You can also right-click on any element in the ARM template Outline to initiate the insert item process.
```
Comment (StephenWeatherford) [inline README.md:52]:
```suggestion
You can also right-click on any element in the ARM Template Outline view to initiate the insert item process.
```

Comment (StephenWeatherford) [reply README.md:52]: Or get rid of both capitals.  This is how it was mentioned above.

**Comment by StephenWeatherford** [comment]
One minor suggestion

File: README.md (hunk 3 above)
Comment (neilpeterson) [reply README.md]: Fixed

**Pull request merged**